content/community/team.md

```diff
@@ -19,6 +19,7 @@ The Kokkos Team is a multi-institutional group of HPC, C++ and Linear Algebra ex
 {{< image src="img/ANL-logo-300x105.png"           class="community-team-image-medium">}}
 {{< image src="img/CSCS-Logo-258X79.jpg"           class="community-team-image-small">}}
 {{< image src="img/LANL-logo-247x89.jpg"           class="community-team-image-large">}}
+{{< image src="img/LBNL-logo.png"                  class="community-team-image-large">}}
```

Comment:
It would be nice if we stuck to the convention above.  @rgayatri23 did the image you found not have that info?